### PR TITLE
Tutorial how to use GitLab Agent

### LINKED ISSUE #113: Tutorial how to use GitLab Agent
Document how to use the GitLab Agent on APPUiO Cloud.

A few hints: https://github.com/appuio/appuio-cloud-community/discussions/41#discussioncomment-2615278

### PR DESCRIPTION
**Pull request by akosma**
Fixes #113 and #111.

## Checklist

- [x] Try to isolate changes into separate PRs (to build a better changelog).
- [x] Categorize the PR by setting a good title and adding one of the labels:
      `how-to`, `tutorial`, `reference`, `explanation`, `cicd`, `dependency`
      as they show up in the changelog
- [x] Link this PR to related issues if applicable.

### CODE REVIEW

File: docs/modules/ROOT/pages/how-to/connect-gitlab.adoc [hunk 1]
```diff
@@ -2,61 +2,174 @@
...
+. Grant namespace admin permissions for the Agent's service account:
++
+[source, shell]
+--
+oc policy add-role-to-user admin -z gitlab-agent --rolebinding-name gitlab-agent
+--
+
+You can now modify your `.gitlab-ci.yml` file to deploy your manifests whenever needed; for example, every time a new tag is pushed:
+
+[source, yaml]
```
Comment (ccremer) [inline docs/modules/ROOT/pages/how-to/connect-gitlab.adoc:61]: I usually put the filename as the title of the code snippet, but no hard feeling.
```suggestion

.gitlab-ci.yml
[source, yaml]
```

File: docs/modules/ROOT/pages/how-to/connect-gitlab.adoc [hunk 2]
```diff
@@ -2,61 +2,174 @@
...
+.GitLab connected to GitLab Agent on APPUiO Cloud
+image::how-to/gitlab-agent.png[]
+
+=== Reusing the GitLab Agent from other Projects
+
+You can reuse this GitLab Agent instance from other projects. Let's imagine that it's installed in a project called `gitlab-agent-test`.
+
+. Create another project:
++
+[source, shell]
```
Comment (ccremer) [inline docs/modules/ROOT/pages/how-to/connect-gitlab.adoc:94]:
```suggestion
[source, bash]
```

File: docs/modules/ROOT/pages/how-to/connect-gitlab.adoc [hunk 3]
```diff
@@ -2,61 +2,174 @@
...
-** API URL: Use the https://portal.appuio.cloud/zones[URL corresponding to your chosen zone].
-** CA Certificate: leave empty.
-** Service Token: paste the value copied in the `oc sa get-token` command above.
-** Check the "RBAC-enabled cluster" option.
-** Uncheck "GitLab-managed cluster" and "Namespace per environment."
-** Leave the "Project namespace prefix (optional, unique)" empty.
-. Click the button btn:[Add Kubernetes cluster]
-
-image::gitlab-kubernetes.png[]
+[source, shell]
```
Comment (ccremer) [inline docs/modules/ROOT/pages/how-to/connect-gitlab.adoc:164]:
```suggestion
[source, bash]
```

**Review by ccremer — CHANGES_REQUESTED** [review]

File: docs/modules/ROOT/pages/how-to/connect-gitlab.adoc [hunk 4]
```diff
@@ -2,61 +2,174 @@
...
+----
+TOKEN=$(oc sa get-token gitlab-ci)
+export KUBECONFIG=gitlab-ci.kubeconfig
+oc login --server=$OPENSHIFT_API_URL --token=$TOKEN
+unset KUBECONFIG
+----
```
[the remark this answers is not in the document]
Comment (akosma) [reply docs/modules/ROOT/pages/how-to/connect-gitlab.adoc:174]: Yes it does!

**Review by ccremer — CHANGES_REQUESTED** [review]
content LGTM, but not tested.
Please cleanup & squash all these `Update docs/modules/ROOT/pages/how-to/connect-gitlab.adoc)` commits into a single one.

**Review by ccremer — APPROVED** [review]
I've allowed myself to squash your commits and rebase.
If you're happy with it, feel free to merge.

**Review by tobru — APPROVED** [review]

**Pull request merged**